src/Flash.tsx

```diff
@@ -28,7 +57,7 @@ const Flash = styled.div<
   }
 
   svg {
-    color: ${props => get(`flashIcon.${props.variant}`)(props.theme)};
+    color: ${props => get(`colors.alert.${variantAliases[props.variant || 'default']}.icon`)(props.theme)};
```

Comment:
I think we can move this logic into the `variants` object and eliminate the need for `variantAliases` :)
```suggestion
```